web/src/components/Button.tsx

```diff
@@ -130,6 +130,9 @@ function getForeground(type: ButtonTypes) {
 function getSize(size: SizeOptions, type: ButtonTypes, isIconOnly: boolean) {
   if (isIconOnly) {
     switch (size) {
+      case 'xs': {
+        return 'min-w-5 min-h-5';
+      }
```

Comment:
Let @Alportan know about this so he ads it to the design system as well :)

Reply:
Is it regarding the min width and height?

Reply:
There is a new XS size!

So yes 😅

Reply:
Aaa, that's true, I completely missed that 🙈 I'll add it as well since I myself had a need for a smaller button sometimes, but never justified to add it due to accessibility. Although if the hitbox is large enough, we should be good 👍

Reply:
This was to stop the slider jumping around when the buttons were hidden but by fixing the height of the component it now works without the new button size. To keep things simpler in both design and implementation I removed the xs size :)